lambda/printlambda.ml

```diff
@@ -536,6 +538,17 @@ let rec lam ppf = function
         Ident.print id (kind str) value_kind k lam arg;
       let expr = letbody body in
       fprintf ppf ")@]@ %a)@]" lam expr
+  | Lmutlet(k, id, arg, body) ->
+      let rec letbody = function
+        | Lmutlet(k, id, arg, body) ->
+            fprintf ppf "@ @[<2>%a =%a@ %a@]"
+              Ident.print id value_kind k lam arg;
+            letbody body
+        | expr -> expr in
+      fprintf ppf "@[<2>(let[mut]@ @[<hv 1>(@[<2>%a =%a@ %a@]"
+        Ident.print id value_kind k lam arg;
+      let expr = letbody body in
+      fprintf ppf ")@]@ %a)@]" lam expr
```

Comment:
Note: here I think the duplication is cumbersome, and could be avoided with a refactoring of the code (an auxiliary function or something).

Reply:
I addressed that in https://github.com/ocaml/ocaml/pull/10090/commits/4600d47e69b3c467d4d421532ab89519f45f63b0.